app/notification_push.go

```diff
@@ -55,6 +56,11 @@ type PushNotification struct {
 func (a *App) sendPushNotificationSync(post *model.Post, user *model.User, channel *model.Channel, channelName string, senderName string,
 	explicitMention bool, channelWideMention bool, replyToThreadType string) *model.AppError {
 	cfg := a.Config()
+	message, errMarkdown := utils.StripMarkdown(post.Message)
+	if errMarkdown != nil {
+		mlog.Warn("failed parse to markdown", mlog.Err(errMarkdown))
+	}
+	post.Message = message
```

Comment:
```suggestion
	message, err := utils.StripMarkdown(post.Message)
	if err != nil {
		mlog.Warn("Failed parse to markdown", mlog.String("post_id", post.Id), mlog.Err(err))
	} else {
	        post.Message = message
	}
```

And rename `err` to `appErr` below.

Reply:
I'm sorry, I miss understanding this one 🙏